specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -4,56 +4,73 @@ import "@azure-tools/typespec-azure-core";
+  @doc("Pass fail criteria for response time metric.") response_time_ms,
+  @doc("Pass fail criteria for response time metric.") latency,
+  @doc("Pass fail criteria for error metric.") error,
+  @doc("Pass fail criteria for total requests.") requests,
   @doc("Pass fail criteria for request rate.") requests_per_sec,
 }
 
+@doc("Aggregation functions for pass/fail criteria.")
 enum PFAgFunc {
-  @doc("Criteria applies for count value") count,
-  @doc("Criteria applies for given percentage value") percentage,
-  @doc("Criteria applies for avg value") avg,
-  @doc("Criteria applies for 50th percentile value") p50,
-  @doc("Criteria applies for 90th percentile value") p90,
-  @doc("Criteria applies for 95th percentile value") p95,
-  @doc("Criteria applies for 99th percentile value") p99,
-  @doc("Criteria applies for minimum value") min,
-  @doc("Criteria applies for maximum value") max,
+  @doc("Criteria applies for count value.") count,
+  @doc("Criteria applies for given percentage value.") percentage,
+  @doc("Criteria applies for avg value.") avg,
+  @doc("Criteria applies for 50th percentile value.") p50,
+  @doc("Criteria applies for 90th percentile value.") p90,
+  @doc("Criteria applies for 95th percentile value.") p95,
+  @doc("Criteria applies for 99th percentile value.") p99,
+  @doc("Criteria applies for minimum value.") min,
+  @doc("Criteria applies for maximum value.") max,
 }
 
+@doc("Action to take on failure of pass/fail criteria.")
 enum PFAction {
-  @doc("Test will continue to run even if pass fail metric criteria metric gets failed")
+  @doc("Test will continue to run even if pass fail metric criteria metric gets failed.")
   continue,
   @doc("Test run will stop if pass fail criteria metric is not passed.") stop,
 }
 
+@doc("Pass/fail criteria result.")
 enum PFResult {
   @doc("Given pass fail criteria metric has passed.") passed,
   @doc("Given pass fail criteria metric couldn't determine.") undetermined,
   @doc("Given pass fail criteria metric has failed.") failed,
 }
 
+@doc("Types of secrets supported.")
 enum SecretType {
-  @doc("If the secret is stored in an Azure Key Vault") AKV_SECRET_URI,
-  @doc("If the Plain text secret value provided") SECRET_VALUE,
+  @doc("If the secret is stored in an Azure Key Vault.") AKV_SECRET_URI,
+  @doc("If the Plain text secret value provided.") SECRET_VALUE,
 }
 
+@doc("Types of certificates supported.")
 enum CertificateType {
-  @doc("If the certificate is stored in an Azure Key Vault") AKV_CERT_URI,
+  @doc("If the certificate is stored in an Azure Key Vault.") AKV_CERT_URI,
 }
 
+@doc("Types of file supported.")
 enum FileType {
-  @doc("If file is jmx script") JMX_FILE,
-  @doc("If file is user properties") USER_PROPERTIES,
-  @doc("If file is not any of other supported type") ADDITIONAL_ARTIFACTS,
+  @doc("If file is jmx script.") JMX_FILE,
+  @doc("If file is user properties.") USER_PROPERTIES,
+  @doc("If file is not any of other supported type.") ADDITIONAL_ARTIFACTS,
```

Comment:
nit: should additional_artifacts be last. Does that change the enum interpretation?

Reply:
I think order doesn't matter if we use it as string.